src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,18 +11,18 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private static String feedbackToUser;
 
     /** The list of persons that was produced by the command */
     private final List<? extends ReadOnlyPerson> relevantPersons;
 
-    public CommandResult(String feedbackToUser) {
-        this.feedbackToUser = feedbackToUser;
+    public CommandResult(String feedback) {
+        setFeedbackToUser(feedback);
```

Comment:
What is the point of this set method? You need to simply assign that value to a private class variable. a set method is not necessary.